src/pages/HorseRace/HorseRace_Logic.tsx

```diff
@@ -8,8 +8,13 @@ const getMockRunners = () => {
   return Array.from({ length: 5 }, () => undefined);
 };
 
+export type ParamsType = {
```

Comment:
should this maybe be called `HorseRaceParams` or something a bit more descriptive?

Reply:
sounds like me